### DISCUSSION TITLE
Impossible to upload proof of education status as flagged by pre-screening process

### DISCUSSION THREAD

**Discussion by ajcoakley** [community/community#28109] [GitHub Education]
I cannot satisfy the requirement to upload proof of enrollment by the pre-screening process.

When I try to upload any document other than my ID, the process informs me other students from my school upload ID and that's what I should do. Our school ID has photo/logo in the front and date on the back. I cannot show both views at the same time.

How should I navigate this issue to successfully provide proof of enrollment? Is there a way to get past this pre-screening stage and to a person?

**Comment by mizzleinetimi** [comment]
i have a similar issue!
